indexing-service/src/test/java/org/apache/druid/indexing/common/task/RowFilterTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.indexing.common.task;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.apache.druid.data.input.InputRow;
+import org.apache.druid.data.input.MapBasedInputRow;
+import org.apache.druid.java.util.common.DateTimes;
+import org.apache.druid.segment.incremental.ThrownAwayReason;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+public class RowFilterTest
+{
+  private static final List<String> DIMENSIONS = ImmutableList.of("dim1");
+
+  @Test
+  public void testFromPredicateAccept()
```

Comment:
Nit: Different style which makes the tested method name more apparent, but it is okay if you prefer the current style itself.


```suggestion
  public void test_fromPredicate_whichAllowsAll()
```